### DISCUSSION TITLE
Error while publishing the space

### DISCUSSION THREAD

**Discussion by JoanDuarte** [GitbookIO/community#307] [0. General bugs and questions] [answered]
This bug is happening when we enable the "publish on the web" feature. How can we solve this? 

<img width="507" alt="Captura de pantalla 2023-08-16 a la(s) 17 17 00" src="https://github.com/GitbookIO/community/assets/85464819/a16e56ab-0392-4d73-94ab-c595a78c681d">

**Comment by aikoii** [comment, marked as answer]
Hey @JoanDuarte! If you could send us the link to the space you're trying to publish at support@gitbook.com, we'll be able to look into it for you. Thanks so much! 🙂
